README.md

```diff
@@ -1,6 +1,55 @@
 # curriculum-cr-training-mobile-menu
 
 
-This repo host an initial code for the tasks performed during the Code Reviewers training.
+> This repo host an initial code for the tasks performed during the Code Reviewers training.
 
 Application is based on this [Figma design](https://www.figma.com/file/t3EJUCAEViw3QasuJLPLVT/Microverse-Student-Potfolio-Templates-Main?node-id=1%3A1471).
+
+This is code review practice project.
+
+## Built With
+
+- HTML
+- CSS
+
+## Live version
+[live version](https://alexander16108.github.io/CR-mobile-menu/https://alexander16108.github.io/CR-mobile-menu/)

```

Comment:
I noticed the Live link to your project is broken. Please, check to confirm what actually is the issue with the link. Having a broken link in your README.md file is unprofessional.